File: backend/src/main/kotlin/com/moa/backend/model/dto/utility/BulkIdeaDto.kt
package com.moa.backend.model.dto.utility

import com.moa.backend.model.slim.IdeaBoxSlimDto
import com.moa.backend.model.slim.IdeaSlimDto

data class BulkIdeaDto (

    var ideaBox: IdeaBoxSlimDto,

    var ideas: MutableList<IdeaSlimDto>
)

File: frontend/src/app/models/dto/utility/bulkIdeaDto.ts
import { IdeaBoxSlimDto } from "../../slimDto/ideaBoxSlimDto";
import { IdeaSlimDto } from "../../slimDto/ideaSlimDto";

export interface BulkIdeaDto {
    ideaBox: IdeaBoxSlimDto;
    ideas: IdeaSlimDto[];
}